packages/api/models/base.js

```diff
@@ -455,7 +459,22 @@ class Manager {
     Object.entries(item).forEach(([fieldName, value]) => {
       if (itemKeyFieldNames.includes(fieldName)) return;
       if (value === undefined) return;
-
+      if (this.allowNulls) {
```

Comment:
This is needed to ensure null values are *removed* and not set to null.       I set it as a object property as it avoids code duplication in case other dynamo models need this.       `this.parseEmptyFilesArrayAsNull` switches the object behavior based on a property as overriding this method in child classes seemed less sensible, particularly given we're removing this in a few revs.

Reply:
Also this approach feels preferable to directly manipulating and passing fieldsToDelete down the call chain as updating the model to respect the mutable fields may be undesirable